templates/prometheus.go

```diff
@@ -34,42 +35,48 @@ var resourceSelector = metav1.LabelSelector{
 	},
 }
 
+const (
+	//TODO: Construct monitoring dns name dynamically based on AWS and IBM Cloud Monitoring regions
+	MonitoringDNSName = "ingest.us-south.monitoring.cloud.ibm.com"
+)
+
 var (
 	KubeRBACProxyPortNumber             int    = 9339
 	PrometheusServingCertSecretName     string = "prometheus-serving-cert-secret"
 	PrometheusKubeRBACPoxyConfigMapName string = "prometheus-kube-rbac-proxy-config"
+	MonitoringURL                       string = fmt.Sprintf("https://%s/prometheus/remote/write", MonitoringDNSName)
```

Comment:
Why do we need to dynamically construct a string here where the entire endpoint (not URL) is static?
Also, this is a remote wWrite endpoint, not a monitoring endpoint.
```suggestion
	remoteWriteEndpoint                     string = fmt.Sprintf("https://ingest.us-south.monitoring.cloud.ibm.com/prometheus/remote/write", MonitoringDNSName)
```

Reply:
This endpoint was being used in a couple of places and so I have asked to use a const.

Reply:
Will make this static for now